src/app/themes/theme.js

```diff
@@ -68,32 +74,40 @@ const theme = createMuiTheme({
       fontSize: '0.875em',
     },
   },
+/*End of Typography*/
+
+/*Start of Overides */
```

Comment:
Is this comment needed? The next line says this is overrides. Comments that don't provide useful information are worse than useless, because they inevitably become obsolete and wrong

Reply:
Why was this conversation resolved?